model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -931,8 +944,8 @@ async def execute(
                 )
 
         if request.inference_framework_image_tag == "latest":
-            request.inference_framework_image_tag = self.docker_repository.get_latest_image_tag(
```

Comment:
i thought we're going to fallback to `get_latest_image_tag` when the configmap doesn't work?

Reply:
I think we can just error if "latest" doesn't exist